.github/workflows/build.yml

```
@@ -164,9 +164,9 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Set up Docker Buildx
         if: steps.web-affected.outputs.web == 'true'
```

actionlint (1.7.11)

[error] key "if" is duplicated in element of "steps" section. previously defined at line:160,col:9

(syntax-check)